documentation/USAGE.rst

```diff
@@ -959,4 +959,9 @@ command in the base directory::
 
    $ conda install --channel conda-forge matplotlib pandas imagemagick
 
+   The PDF file requires `SIunits.sty`_ available, e.g., from
+   `texlive-science`_.
+
 .. _download the latest manual:  http://www.ctcms.nist.gov/fipy/download/
+.. _SIunits.sty: https://ctan.org/pkg/siunits
```

Comment:
Per the linked [CTAN](https://ctan.org/pkg/siunits) page, SIunits is superseded by [SIunitx](https://ctan.org/pkg/siunitx). It's available for both TeXLive (Linux/MacOS) and MiKTeX (Windows).

Reply:
I'm aware. It's presumably different in some way (if it's not different in some way, why did they "supercede" it?!?). Since `SIunits.sty` can still be obtained, I didn't feel that this particular pull request needed to delve into switching.

Reply:
Since SIunits is in maintenance mode, not entirely deprecated, I guess it's OK? Feels like pushing technical debt onto our future-selves, though.